src/accordion/accordion.ts

```diff
@@ -216,7 +239,18 @@ export class NgbAccordion implements AfterContentChecked {
    */
   @Output() panelChange = new EventEmitter<NgbPanelChangeEvent>();
 
-  constructor(config: NgbAccordionConfig) {
+  /**
+   * An event emitted when a panel is shown, after the transition. The payload is the panel id.
+   */
+  @Output() shown = new EventEmitter<string>();
+
+  /**
+   * An event emitted when a panel is hidden, after the transition. The payload is the panel id.
```

Comment:
Same here, but `collapsing animation`. Would also be nice to precise, if DOM content is expected to be removed at this point or not.